packages/template-chakra-storefront/src/pages/product-list/partials/empty-results.jsx

```diff
@@ -32,74 +57,41 @@ const EmptySearchResults = ({searchQuery, category}) => {
         >
             <SearchIcon boxSize={[6, 12, 12, 12]} marginBottom={5} />
             {!searchQuery ? (
-                <Fragment>
-                    {' '}
-                    <Text fontSize={['l', 'l', 'xl', '2xl']} fontWeight="700" marginBottom={2}>
-                        {intl.formatMessage(
-                            {
-                                id: 'empty_search_results.info.cant_find_anything_for_category',
-                                defaultMessage:
-                                    'We couldn’t find anything for {category}. Try searching for a product or {link}.'
-                            },
-                            {
-                                category: category?.name,
-                                link: (
-                                    <Link as={RouteLink} to={'/'}>
-                                        {intl.formatMessage(contactUsMessage)}
-                                    </Link>
-                                )
-                            }
-                        )}
-                    </Text>{' '}
-                </Fragment>
+                <Text fontSize={['l', 'l', 'xl', '2xl']} fontWeight="700" marginBottom={2}>
+                    {messages.cantFindAnythingForCategory({
+                        category: category?.name,
+                        link: (
+                            <Link as={RouteLink} to={'/'}>
+                                {messages.contactUs}
+                            </Link>
+                        )
+                    })}
+                </Text>
             ) : (
                 <Fragment>
                     <Text fontSize={['lg', 'lg', 'xl', '3xl']} fontWeight="700" marginBottom={2}>
-                        {intl.formatMessage(
-                            {
-                                id: 'empty_search_results.info.cant_find_anything_for_query',
-                                defaultMessage: 'We couldn’t find anything for "{searchQuery}".'
-                            },
-                            {
-                                searchQuery: searchQuery
-                            }
-                        )}
+                        {messages.cantFindAnythingForQuery({
+                            searchQuery: searchQuery
+                        })}
```

Comment:
I think the benefit in readability is not worth the cost of new abstraction. For example, we can do it the React Intl way: 
```suggestion
                        {intl.formatMessage(messages.cantFindAnythingForQuery, {
                            searchQuery: searchQuery
                        })
```